config/migrations/2023/20230217113331-add-missing-betrokken-besturen-kerkfabriek-st-pieter-van-puurs-sint-amands.ttl

```diff
@@ -0,0 +1,7 @@
+# Adding missing betrokken besturen kerkfabriek St.-Pieter van Puurs-Sint-Amands
+
+<http://data.lblod.info/id/betrokkenLokaleBesturen/635237108DE5818A7C9F41D1>
```

Comment:
We are missing the link between the municipality and the betrokkenLokaleBestuur here no ? If it's not there, it'll have troubles dispatching ?
And talking about dispatch, did you check if some submissions would need to be re-dispatched manually with this new link added ?

Reply:
> And talking about dispatch, did you check if some submissions would need to be re-dispatched manually with this new link added ?

No not yet, how do I check this ?

Reply:
Ok thanks, I'll check that :crossed_fingers: